packages/reports/tests/integration/components/report-view-test.js

```diff
@@ -117,30 +147,38 @@ module('Integration | Component | report view', function(hooks) {
     assert.expect(2);
 
     this.set('report.request', {
-      logicalTable: {
-        table: 'network',
-        timeGrain: 'day'
-      },
-      metrics: [{ metric: 'adClicks' }],
-      dimensions: [],
-      filters: [],
-      sort: [
+      table: 'network',
+      dataSource: 'bardOne',
+      limit: null,
+      requestVersion: '2.0',
+      filters: [
+        {
+          type: 'timeDimension',
+          dataSource: 'bardOne',
+          field: 'network.dateTime',
+          parameters: { grain: 'day' },
+          operator: 'bet',
+          values: ['11-04-2020', '11-06-2020']
+        }
+      ],
+      columns: [
+        {
```

Comment:
```suggestion
        {
           type: 'timeDimension',
           source: 'bardOne',
           field: 'network.dateTime',
           parameters: { grain: 'day' },
        },
        {
```

Reply:
When adding the timedimension in we lose the metric label capability thats required for this test

Reply:
Ah ok then it looks like you also need to update the filter values to be what they were before `['current', 'next']`